core/nginx/conf/nginx.conf

```diff
@@ -317,9 +317,11 @@ mail {
       ssl_certificate /certs/letsencrypt/live/mailu/fullchain.pem;
       ssl_certificate /certs/letsencrypt/live/mailu-ecdsa/fullchain.pem;
       {% endif %}
+      {% if TLS_PERMISSIVE == 'true' %}
```

Comment:
This effectively disables TLSv1 and TLSv1.1 on a default config.
Please use:
`{% if TLS_PERMISSIVE %}`
and add this to config.py:
`args['TLS_PERMISSIVE'] = str(args.get('TLS_PERMISSIVE')).lower() not in ('false', 'no')`

This matches the code used in the admin container and will be replaced by the new environment normalizer to be done by me.

Reply:
that's [85a2aaf](https://github.com/Mailu/Mailu/pull/2450/commits/85a2aafcdfd6acf336d9188420519056040dd382)